### DISCUSSION TITLE
Bug: On clicking milestone tag in Boards it doesn't filter the issues correctly

### DISCUSSION THREAD

**Discussion by anadi2311** [community/community#6883] [Projects and Issues]
I have 3 issues with milestones in (In Progress)
![image](https://user-images.githubusercontent.com/47948789/139200049-e1904f1a-9a27-451c-803b-cb9d5c627d30.png)

When I click "nft market Cap MVP" Nothing shows up
![image](https://user-images.githubusercontent.com/47948789/139200177-4a2d5897-c9fc-4541-a41d-5c333a7c38ff.png)

![image](https://user-images.githubusercontent.com/47948789/139200143-7ebc421c-4e10-4b11-8c51-62b9dc568fb0.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
